data/benefits/aide-au-bafa-pour-une-session-de-formation-générale.yml

```
@@ -0,0 +1,30 @@
+label: Aide au Bafa pour une session de formation générale (CAF du Val de Marne)
```

```suggestion
label: aide au Bafa pour une session de formation générale (CAF du Val de Marne)
```

Même commentaire qu'ici https://github.com/betagouv/aides-jeunes/pull/1948#discussion_r760945399/